page.go

```diff
@@ -37,8 +43,8 @@ type Page interface {
 	// Overwrite page content with new content. making sure to trigger before and
 	// after write events.
 	Write(Markdown) bool
-	// ModTime Return the last modification time of the underlying file
-	ModTime() time.Time
+	// ModTime returns the last modification time of the page. If the field Date in metadata is set, this is return, otherwise the modification time of the underlying file.  if real is true, the metadata Date field is ignored
+	ModTime(bool) time.Time
```

Comment:
Can you explain why we need this boolean? why for example it doesn't work like Title?